src/persist/updatableFileAdapter.ts

```diff
@@ -14,7 +14,7 @@ export class UpdatableFileAdapter extends FileAdapter implements UpdatableAdapte
     super(filePath);
   }
 
-  // updatePolicies adds policy rules to the storage.
+  // updatePolicy updates a policy rules to the storage.
```

Comment:
`a policy rules` is wrong English, how is Golang code written here? Actually you can just copy from Go comments? Why don't do this?

Reply:
these comments were copied fron the addpolicies, maybe i forget to change

Reply:
Use Golang Casbin as a reference, including the comments. This is mandatory.